mapdata/WarcraftLegacies/BuffData/Skin/1211379778.json

```diff
@@ -0,0 +1,29 @@
+{
+  "OldId": 1835619650,
+  "NewId": 1211379778,
+  "Unk": [
+    0
+  ],
+  "Modifications": [
+    {
+      "Id": 1885959270,
+      "Type": 3,
+      "Value": "Torrent"
+    },
+    {
+      "Id": 1700951398,
+      "Type": 3,
+      "Value": "This unit has been torrented; it is in the air for a short duration."
```

Comment:
Your attention to detail here is legendary, but I don't think we want a custom buff for this one. We should just use Stunned, since it's a stun.